packages/shared/src/utils/constants/phone-number-verification.ts

```diff
@@ -0,0 +1,4 @@
+export const VERIFICATION_SERVICES = {
+    SMS: 'SMS',
```

Comment:
can we make its `SMS: 'sms'` since the api expects payloads in small caps and we use these const inside`useGetPhoneNumberOTP` hook 🙏 likewise for whatsapp, thank you

Reply:
I was thinking of adding this way for display as from the API side you just have to add `.ToLowerCase()`. So we don't have to implement a lot of logics for the string `WhatsApp` in the component base 🙏 what do you think?

Reply:
This works as well! Thanks